SimPEG/VRM/ProblemVRM.py

```diff
@@ -520,7 +771,7 @@ def fields(self, m):
 
         # Project to active mesh cells
         # m = np.matrix(self.xiMap * m).T
-        m = np.matrix(self.xi).T
```

Comment:
here, it is preferable to work with `self.xi` rather than the mapping (asking for `self.xi` will perform the mapping). This way users can either specify a fixes `xi` or one which is active in the inversion by providing a mapping